src/icalendar/cal/component.py

```diff
@@ -663,5 +669,89 @@ def new(
         component.comments = comments
         return component
 
+    def to_jcal(self) -> list[str, list, list]:
+        """Convert this component to a jcal object.
+
+        Returns:
+            jcal object
+
+        See also :attr:`to_json`.
+        """
+        properties = []
+        for key, value in self.items():
+            for item in value if isinstance(value, list) else [value]:
+                properties.append(item.to_jcal(key.lower()))
+        return [
+            self.name.lower(),
+            properties,
+            [subcomponent.to_jcal() for subcomponent in self.subcomponents],
+        ]
+
+    def to_json(self) -> str:
+        """Return this component as a jCal JSON string.
+
+        Returns:
+            JSON string
+
+        See also :attr:`to_jcal`.
+        """
+        return json.dumps(self.to_jcal())
+
+    @classmethod
+    def from_jcal(cls, jcal: str | list) -> Component:
+        """Create a component from a jcal object.
+
+        Returns:
+            Component
+
+        Raises:
+            JCalParsingError: If the jcal provided is invalid.
+            JSONDecodeError: If the string provided not valid JSON.
```

Comment:
It's interesting to note that `JCalParsingError` links in https://icalendar--979.org.readthedocs.build/en/979/reference/api/icalendar.cal.component.html#icalendar.cal.component.Component.from_jcal, but `JSONDecodeError` does not. Can you try to find a fix to make the latter link?

I see it's in the Python standard library at https://docs.python.org/3/library/json.html#json.JSONDecodeError.

Perhaps one of these two will work? See also from https://www.sphinx-doc.org/en/master/usage/domains/python.html#role-py-exc.

```
            json.JSONDecodeError: If the string provided not valid JSON.
            :py:exc:`json.JSONDecodeError`: If the string provided not valid JSON.
```